lib/nip44/nip44_v2.dart

```diff
@@ -10,7 +10,7 @@ import 'package:cryptography/cryptography.dart' as cryptography;
 // code from https://github.com/0xchat-app/nostr-dart
 class NIP44V2 {
   static final _digest = SHA256Digest();
-  static final _hmacBlockSize = 64;
+  static const _hmacBlockSize = 64;
```

Comment:
sorry if this is a dumb question, but why const over final? @bryanmontz

Reply:
`const` was recommended by the IDE because this is a compile-time constant. Compile-time constants get further optimizations that `final` doesn't provide. `final` is similar but is immutable *after initialization*. `const` is deeply immutable.